backend/e4h-services/ingestion-service/app/api/endpoints/template_generation.py

```
@@ -327,11 +328,35 @@ async def get_facility_QR_for_autologin(
-            # Generate QR
-            img = qrcode.make(login_url)
+            qr = qrcode.make(login_url).convert("RGB")
+
+            # Create a new image (taller) to hold QR and text
+            width, height = qr.size
+            font_size = 30
+            padding = 10
+
+            try:
+                font = ImageFont.truetype("DejaVuSans-Bold.ttf", font_size)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)